FastSimulation/TrackingRecHitProducer/plugins/PixelBarrelTemplateSmearerPlugin.cc

```diff
@@ -0,0 +1,128 @@
+
```

Comment:
I doubt there is need for 2 implementations
PixelBarrelTemplateSmearerPlugin and
PixelForwardTemplateSmearerPlugin

I think that just one plugin will do and it can enherrit directly from trackingRecHitAlgorithm.
I.e. PixelTemplateSmearer should, after some small changes, become the plugin.